ompi/mca/mtl/psm2/mtl_psm2_component.c

```diff
@@ -323,6 +323,18 @@ ompi_mtl_psm2_component_open(void)
 static int
 ompi_mtl_psm2_component_query(mca_base_module_t **module, int *priority)
 {
+
+    /*
+     * Mixing the PSM2 MTL with the OFI BTL (using PSM2) 
+     * can cause an issue when they both call psm2_finalize
+     * in older versions of libpsm2.
+     */
+    if (!psm2_get_capability_mask(PSM2_LIB_REFCOUNT_CAP)) {
+        opal_output_verbose(2, ompi_mtl_base_framework.framework_output, 
+            "Please install version 11.2.173 version or later of the libpsm2 library.\n");
```

Comment:
A suggestion (which you are free to use or not):

> The PSM2 MTL component will not work unless you have at least version 11.2.173 of the libpsm2 library installed.

This slightly changes the tone of the message to be: if you want to use the PSM2 MTL, then you have to install this library (vs. "you must install this library, no matter what you're doing").

Reply:
I changed it to

> This version of the PSM2 MTL needs version 11.2.173 or later of the libpsm2 library for correct operation.